frontend/amundsen_application/api/utils/search_utils.py

```diff
@@ -38,6 +45,22 @@ def map_table_result(result: Dict) -> Dict:
     }
 
 
+def map_feature_result(result: Dict) -> Dict:
+    return {
+        'type': 'feature',
+        'description': result.get('description', None),
+        'key': result.get('key', None),
+        'last_updated_timestamp': result.get('last_updated_timestamp', None),
+        'name': result.get('feature_name', None),
+        'feature_group': result.get('feature_group', None),
+        'version': result.get('version', None),
+        'availability': result.get('availability', None),
+        'entity': result.get('entity', None),
+        'badges': result.get('badges', None),
+        'status': result.get('status', None),
```

Comment:
For fields like badges and availability, should we use `[]` instead of `None`?

Reply:
following convention from table equivalent